projects/hitl/software/artifacts/report.html

```diff
@@ -0,0 +1,340 @@
+<!DOCTYPE html>
```

Comment:
Not sure if we want auto-generated reports in our code? Might make merge conflicts annoying if we keep writing to the same place. When we Bazelify this project, this won't be an issue because Bazel will track everything in it's folders. We should chat about Bazel...